modules/ossm-install-kiali-operator.adoc

```diff
@@ -0,0 +1,52 @@
+// Module included in the following assemblies:
+//
+// * service-mesh-docs-main/kiali/ossm-kiali-assembly.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="ossm-install-kiali-operator_{context}"]
+= Installing the {KialiProduct}
+:context: ossm-install-kiali-operator
+
+// TP 1 Content
+// Possible file name may change
+// Possible assembly file may change
+// Assemblies, topic map info needs to be worked out still for 3.0.
+// Kiali install content for TP1 is the same as install content for 2.x.
+
+The following steps show how to install the {KialiProduct}.
+
+[WARNING]
+====
+Do not install Community versions of the Operators. Community Operators are not supported.
+====
+
+.Prerequisites
+
+* Access to the {product-title} web console.
+
+.Procedure
+
+. Log in to the {product-title} web console.
+
+. Navigate to *Operators* -> *OperatorHub*.
+
+. Type *Kiali* into the filter box to find the {KialiProduct}.
+
+. Click the *{KialiProduct}* to display information about the Operator.
+
+. Click *Install*.
+
+. On the *Operator Installation* page, select the *stable* Update Channel.
+
+. Select *All namespaces on the cluster (default)*. This installs the Operator in the default `openshift-operators` project and makes the Operator available to all projects in the cluster.
+
+. Select the *Automatic* Approval Strategy.
++
+[NOTE]
+====
+The Manual approval strategy requires a user with appropriate credentials to approve the Operator install and subscription process.
```

Comment:
Nit - per the [ISG word usage](https://www.ibm.com/docs/en/ibm-style?topic=word-usage#i) entries on "install" and "installation", install should only be used as a verb, whereas installation should be the choice for nouns and adjectives:
```suggestion
The Manual approval strategy requires a user with appropriate credentials to approve the Operator installation and subscription process.
```